build/darwin/ios_app.gni

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+template("ios_app") {
```

Comment:
can we add an `if (is_ios)` here and assert if we're not ios and trying to use this template?

Reply:
Good call. You can also just say `assert(is_ios)`

Reply:
I think `assert(is_ios)` is fine, but we can't do what I think @gw280 is asking - we need to use this template from iOS.